web_external/InfoView/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+
+import './style.styl';
+
+class InfoView extends PureComponent {
+    constructor(props) {
+        super(props);
+    }
+    render() {
+        return <div className={['v-infoview', this.props.className].join(' ')}>
+            <div className='panel panel-default'>
+                <div className='panel-heading'>Info</div>
+                <div className='panel-body'>
+                    {this.props.annotations && this.props.annotations.length &&
+                        <div>
+                            <ul className='geometry'>
+                                {this.props.annotations.map((annotation) => {
+                                    return <li key={annotation.geometry.id0} className='track'>
+                                        <div>
```

Comment:
Is this `<div>` necessary?

Reply:
There are two options for this situation in React, array with keys, or having a wrapping container.